tools/op.sh

```diff
@@ -347,6 +348,56 @@ function op_clip() {
   op_run_command tools/clip/run.py $@
 }
 
+function op_ui() {
+  op_before_cmd
+
+  if [[ "$1" == "help" || "$1" == "--help" ]]; then
+    echo "Usage: op ui <command> [args]"
+    echo ""
+    echo -e "${BOLD}${UNDERLINE}Commands [Display]:${NC}"
+    echo -e "  ${ITALICS}[none]${NC}     Launch the UI at the main screen ${ITALICS}(default)${NC}"
+    echo -e "  ${BOLD}setup${NC}      Launch the UI at the setup device screen"
+    echo -e "  ${BOLD}reset${NC}      Launch the UI at the reset device screen"
+    echo -e "  ${BOLD}updater${NC}    Launch the UI at the update device screen"
+    echo ""
+    echo -e "${BOLD}${UNDERLINE}Commands [Utility]:${NC}"
+    echo -e "  ${BOLD}build${NC}      Build the UI"
+    echo -e "  ${BOLD}test${NC}       Run the UI tests"
+    echo -e "  ${BOLD}help${NC}       Show this help message"
+    return 0
+  fi
+
+  UI_DIR=selfdrive/ui
+  if [[ "$1" == "build" ]]; then
+    shift
+    NO_VERIFY="1" op_build $UI_DIR $@
+  elif [[ "$1" == "test" ]]; then
+    shift
+    NO_VERIFY="1" op_test $UI_DIR $@
+  else
+    # Get the UI binary to run
+    UI_BIN="$UI_DIR/ui"
+    if [[ "$1" == "setup" || "$1" == "reset" || "$1" == "updater" ]]; then
+      CMD=$1; shift
+      UI_BIN="$UI_DIR/qt/setup/$CMD"
```

Comment:
these are moving to system/ui with raylib, so this will break shortly

Reply:
Absolutely. I noticed that, but it didn't seem like they were currently implemented. The benefit is you can easily change the underlying command while keeping `op ui` command the same.